### PR TITLE
PIM-5788: Test migration scripts from 1.5 to 1.6 and proof read the UPGRADE.md

### PR DESCRIPTION
**Pull request by aRn0D**

### CODE REVIEW

File: UPGRADE-1.6.md [hunk 1]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+         cd $PIM_DIR
+         composer update
+        ```
+
+        This step will also copy the upgrades folder from `vendor/akeneo/pim-community-dev/` to your Pim project root to allow you to migrate.
+
+8. Then you can migrate your database using:
+
+    ```
+     php app/console doctrine:migration:migrate
```
Comment (aRn0D) [inline UPGRADE-1.6.md]: ??
```
* Add a the parameter `doctrine_migrations_dir` to `parameter.yml`:
    
        ```
         doctrine_migrations_dir: %kernel.root_dir%/../upgrades/1.5-1.6
        ```
```

**Comment by nidup** [comment]
GTM

**Comment by loicguillois** [comment]
GTM

File: UPGRADE-1.6.md [hunk 2]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+     export PIM_DIR=/path/to/your/pim/installation
+     cp app/SymfonyRequirements.php $PIM_DIR/app
+     cp app/SymfonyRequirements.php $PIM_DIR/app
+     cp app/config/config.yml $PIM_DIR/app/config/ should be cp app/config/pim_parameters.yml $PIM_DIR/app/config ?
+     cp composer.json $PIM_DIR/
+    ```
+
+3. Update your **config.yml**
+
+    * Remove the configuration of the `pim_catalog` from this file.
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: don't understand what it means

File: UPGRADE-1.6.md [hunk 3]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+
+4. Update your **app/AppKernel.php**:
+
+    * Remove the following bundles: `PimBaseConnectorBundle`, `PimTransformBundle`, `NelmioApiDocBundle`
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: does not make sense, you made me download pim-community-standard-v1.6-latest.tar.gz
the AppKernel does not contain that

File: UPGRADE-1.6.md [hunk 4]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+
+5. Update your **app/config/routing.yml**: 
+
+    * Route removed: `nelmio_api_doc`
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: non consistent with what's above

File: UPGRADE-1.6.md (hunk 3 above)
Comment (jjanvier) [inline UPGRADE-1.6.md]: take example here, it's much more detailed on how to do https://github.com/akeneo/pim-community-dev/blob/1.5/UPGRADE-1.5.md

File: UPGRADE-1.6.md (hunk 4 above)
Comment (jjanvier) [inline UPGRADE-1.6.md]: example here on how to do https://github.com/akeneo/pim-community-dev/blob/1.5/UPGRADE-1.5.md

File: UPGRADE-1.6.md [hunk 5]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+    * **Caution**, don't forget to re-add your own dependencies to your *composer.json* in case you have some:
+        
+        ```
+        "require": {
+            "your/dependencies": "version",
+            "your/other-dependencies": "version",
+        }
+        ```
+        
+        Especially you store your product in Mongo, don't forget to add `doctrine/mongodb-odm-bundle`:
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: if

File: UPGRADE-1.6.md [hunk 6]
```diff
@@ -190,55 +190,74 @@ See the documentation [here](http://docs.akeneo.com/latest/reference/import_expo
...
+        
+        ```
+        "require": {
+            "doctrine/mongodb-odm-bundle": "3.2.0"
+        }
+        ```
+    
+    * Then run the command:
```
Comment (jjanvier) [inline UPGRADE-1.6.md]: to update your dependencies

File: upgrades/1.5-1.6/Version_1_6_20160804181154_versioning_index.php [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        $this->addSql('CREATE INDEX logged_at_idx ON pim_versioning_version (logged_at)');
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```
Comment (jjanvier) [inline upgrades/1.5-1.6/Version_1_6_20160804181154_versioning_index.php]: do like other migrations 
throw an exception

**Comment by jjanvier** [comment]
I don't understand why you moved scripts to the folder upgrades/1.5-1.6
(like it will not work with doctrine migrate)

but maybe it's not normal regarding what you want to do later

File: UPGRADE-1.6.md (hunk 3 above)
Comment (aRn0D) [inline UPGRADE-1.6.md]: Personally I disagree !

File: UPGRADE-1.6.md (hunk 4 above)
Comment (aRn0D) [inline UPGRADE-1.6.md]: Don't understand !

**Pull request merged**